test/performance/io/detail/view_take_until_benchmark.cpp

```diff
@@ -46,7 +46,13 @@ void sequential_read(benchmark::State & state)
         {
             single_pass_or_ref_t single_pass_or_ref{container};
             for (auto elem : single_pass_or_ref)
+            {
                 sum += elem;
+                if (elem >= 100)
+                {
+                    break;
+                }
+            }
```

Comment:
```suggestion
            for (auto elem : single_pass_or_ref)           
                if (sum += elem ; elem >= 101)
                    break;
```

Less code :)

I think it should be `101` (from 3.0.2):

https://github.com/seqan/seqan3/blob/370673ffd20d790f759aed15771072436a422ec3/test/performance/range/views/view_take_until_benchmark.cpp#L42-L44

Reply:
Oh yeah, it should be 101....I can't even copy&paste xD